Video 6 - draft script (#14)
asyncapi/training: VIDEO_6_SCRIPT.md

# Video 6: Working in community

- What and why working within a community
- AsyncAPI and its community
- How to join the AsyncAPI community

Hello everybody! I hope that you’re doing well. Welcome again to the AsyncAPI video, where we explain the project and its progress. Today we’ll talk about the AsyncAPI community.

> ## What and why working within a community

Reviewer: you should try this text as a potential lector for the video 😛 

it has many difficult words, talks about many nice things but I have the impression it is overcomplicating the message 🤔 

why not, in super simple words 😄 , just write "that community is about gathering people with the same goals and vision to work as equals together, collaborate and drive the change at scale much grater that they could as individuals"?

it is to serious now, not good for video I think

But, nowadays, it's not necessary to share the same time and space to create a group of people committed to working on a project. Today the number of virtual communities is growing exponentially, letting people from all over the world exchange knowledge and ideas.

We can define three main characteristics of virtual communities:

•	They allow communication between users, either synchronously or asynchronously.
•	The information can be exchanged in different formats, including images, videos, documents, audio...
•	They promote meaningful, collaborative, and team-based learning.

Beyond generating a sense of belonging, communities facilitate the exchange of information, knowledge, and experiences, contributing to their members' personal and professional development. Virtual communities are, nowadays, increasingly essential for professional and, as well, social.

Human beings use somehow virtual communities to tackle and reverse the problems of physical communities virtual ones are very valuable not when shortening geographical and cultural distances, but also to restore the lack of social integration of groups discriminated within real communities.

> ## AsyncAPI and its community

Reviewer: ```suggestion
## AsyncAPI Initiative and its community
```

> As mentioned in previous videos, AsyncAPI is a specification managed and driven by its community. It’s a virtual community open to everyone who participates in the specification. Everyone is welcome. AsyncAPI is devoted to forming a community with the most accurate possible reflection of contemporary society, diverse and inclusive.

Reviewer: ```suggestion
As mentioned in previous videos, AsyncAPI Initiative provides spec and tools managed and driven by its community. It’s a virtual community open to everyone who participates in the specification. Everyone is welcome. AsyncAPI is devoted to forming a diverse and inclusive community.
```

less difficult words that are hard to pronounce 😆

> Being a virtual community comprises many people from a different races, ethnicities, and cultures. As the community grows and more and more people get in, it’s increasingly broad and diverse.

Reviewer: you mentioned `virtual` few times, but it is a first time I hear people talk about community as virtual community. Not sure if it makes sense to include this phrase

# Video 6: Working in community

> Hello everybody! I hope that you’re doing well. Welcome again to the AsyncAPI video, where we explain the project and its progress. My name is ... Today we’ll talk about the AsyncAPI community. We will tackel the 

Reviewer: ```suggestion
Hello everybody! I hope that you’re doing well. Welcome again to the AsyncAPI video, where we explain the project and its progress. My name is ... Today we’ll talk about the AsyncAPI community. We will tackle the 
```

there was a tiny spelling mistake

## Working as a community

A community is united by goals projected for the future, hoping to build something valuable. The projection of a reachable better place to be. Even slowly, if it goes far, as the italians say. 

Beyond generating a sense of belonging, communities facilitate the exchange of information, knowledge, and experiences, contributing to their members' personal and professional development. 

The communities are here to tackle and reverse the problems of physical ones. They somehow are very valuable not when shortening geographical and cultural distances, but also to restore the lack of social integration of groups discriminated within real communities. They are open and inclusive by definition.

## AsyncAPI Initiative and its community

As mentioned in previous videos, AsyncAPI Initiative provides spec and tools managed and driven by its community. It’s a community open to everyone who participates in the specification. Everyone is welcome. AsyncAPI is devoted to forming a diverse community, comprising many people from different races, ethnicities, and cultures.

> That's why AsyncAPI regularly participate in different mentoring programs to onboard open source unexperienced people to the project, to show commitment to mentorship and making community more and more inclusive.

Reviewer 2: **programs dedicated to onboarding inexperienced people to the project and making the community more inclusive**.

> It’s mainly made up of developers, but it’s open to receiving all kinds of professionals who can contribute or collaborate with the project. Everyone has something to say. All kinds of ideas, opinions, inputs, or contributions are welcome. At the same time, all constructive criticism is taken into account to continue growing and improving.

Reviewer 2: **The majority of the AsyncAPI community members are developers, but we remain open to...**

> AsyncAPI intends that everyone who arrives feels welcome, respected, and included in any project proposal, idea, or decision-making. It’s intended that the people who make up the community feel that the project is part of them, generating a sense of belonging.

Reviewer 2: **intends to make everyone feel welcomed, respected...**

AsyncAPI is its community; the community is the engine that manages, maintains, and drives the specification. Summing up, the community is about gathering people with the same goals and vision to work as equals together, collaborate and drive the change at a much greater scale than it could be achieved as individuals.

## How to join the AsyncAPI community

AsyncAPI mainly relies on accepting difference as a virtue and valuing trust in people to construct a solid and coherent community. Feeling comfortable, welcome, trustworthy is the only way to create a sense of belonging. This combination is probably the secret to the constant growth of the community and of the project itself.

If you’re not yet part of our community, we’re looking forward to it. We invite you to participate and enjoy yourself with us.

The steps to joining our community are very simple: follow us on our social networks, LinkedIn, Twitter, and Youtube.

Join AsyncAPI's Slack, GitHub, and visit our website. We look forward to hearing from you and seeing how we can make the project better.

## Farewell

Thank you very much for coming along. I hope you enjoy the ride. 

> We look forward to seeing you in the following video, don't miss it! Have a nice day!

Reviewer 2: You talked about the benefits of an OS community. I think that to make your case stronger about why AsyncAPI chooses to be an OSS driven by its community over other models you need to consider:

- Advantages and disadvantages of private software.
- The disadvantages of an OS community. 

Otherwise, your audience can be left wondering "If OSS communities are so wonderful, why are there successful private software companies like Microsoft?"
